platform/pom.xml

```diff
@@ -159,16 +159,6 @@
             <artifactId>asm</artifactId>
             <version>7.1</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-hdfs-client</artifactId>
-            <version>3.3.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
-            <version>2.10.5.1</version>
-        </dependency>
         <dependency>
```

Comment:
Removed because delta standalone reader isn't compatible with 2.10.5.1: it uses an older version ( 2.8.4) of `jackson-databind` instead. Not sure if this is the right thing to do.